crates/types/src/services/executor.rs

```diff
@@ -264,8 +264,10 @@ pub enum Error {
     OutputAlreadyExists,
     #[display(fmt = "The computed fee caused an integer overflow")]
     FeeOverflow,
-    #[display(fmt = "The computed gas caused an integer overflow")]
-    GasOverflow,
+    #[display(
+        fmt = "The computed gas caused an integer overflow. reason: {_0}, Augend: {_1}, Addend: {_2}"
```

Comment:
Is it possible to inform about the limit (maybe for the user to retry within the limit)

Reply:
the `Addend` should take care of that, in the case of `tx_max_gas > remaining_gas_limit`. maybe it is not too readable.

Reply:
how does this work a532cc8b83 ?

Reply:
Yeah find it more readable like this